docs/dev/installer.md

```diff
@@ -63,24 +63,25 @@ Accepted commands:
 
 Accepted options:
         [--debug]                               Prints additional messages in the console
+        [--extensions-rbac]                     Enable ClusterRole aggregation for easier management of extensions RBAC
+        [--external-logs <logs-provider-url>]   External url to fetch logs from when not available in the cluster
```

Comment:
I feel like this sentence doesnt read right: `External url to fetch logs from when not available in the cluster`

Better? `External url from which to fetch logs, when logs are not available in the cluster`